README.md

```diff
@@ -35,41 +35,27 @@ This is OperationCode.org's repository for their new front-end implementation us
 
 ## Install
 
-- `npm start` - This begins the development server.
+### Development Build
+- `npm install -g yarn` - Install Yarn Globally
```

Comment:
Can you include what 'npm' is and how to install it for mac/windows (if it's not installed by default)